[esx_addons]/esx_property/locales/fr.lua

```diff
@@ -30,8 +30,8 @@ Locales['fr'] = {
   ['press_to_exit'] = 'appuyez sur [E] pour sortir de la propriété',
   ['not_enough'] = 'vous n\'avez pas assez d\'argent',
   ['invalid_quantity'] = 'quantité invalide',
-  ['paid_rent'] = 'you paid your rent at $%s for %s',
-  ['paid_rent_evicted'] = 'you have been ~o~evicted from %s for not affording rent at $%s',
+  ['paid_rent'] = 'vous avez payé votre loyer à $%s pour %s',
```

Comment:
"Vous avez payé votre loyer **de** XX$ pour XXX"
Determinant "De" seems more appropriate in this situation